platform/osgi-platform/src/main/java/org/motechproject/server/osgi/status/PlatformStatusManagerImpl.java

```diff
@@ -20,13 +25,34 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(PlatformStatusManagerImpl.class);
 
+    private static final String EVENT_SUBJECT = "All bundles started.";
+    private static final String ERROR_MESSAGE = "OsgiEventProxy is set null";
```

Comment:
1) Rename the variable so that it indicates what error this talks about - since the message is specific to OsgiEventProxy being null
2) set -> set to